scripts/lib/emoji_utils.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import division, print_function
+import hashlib
+import os
+import shutil
+import sys
+import xml.etree.ElementTree as ET
+from six import unichr
+
+if False:
+    from typing import Dict, Union, Text
+
+def get_success_stamp(EMOJI_CACHE_PATH, EMOJI_PATH):
```

Comment:
I am not sure it's a good style to make the function parameters look like constants. We should probably just make them lower case.